example/azure/build-examples.yaml

```diff
@@ -0,0 +1,138 @@
+parameters:
+  - name: system
+    type: string
+  - name: XAPIEMAIL
+    type: string
+  - name: BUILDINGID
+    type: string
+  - name: BUILDNUMBER
+    type: string
+    default: "202430015"
+  - name: MAPVIEWERDOMAIN
+    type: string
+
+steps:
+    - ${{ if contains(parameters.system, 'macOS') }}:
+      - task: DownloadSecureFile@1
+        name: api_key
+        inputs:
+          secureFile: api_key.json
+
+      - task: InstallAppleCertificate@2
+        inputs:
+          certSecureFile: AppleCert23.p12
+
+      - task: InstallAppleCertificate@2
+        inputs:
+          certSecureFile: DistributorWildcard.p12
+
+      - task: InstallAppleProvisioningProfile@1
+        inputs:
+          provisioningProfileLocation: 'secureFiles'
+          provProfileSecureFile: Situm_Cordova_SDK_Example.mobileprovision
+          removeProfile: true
+
+      - task: InstallAppleProvisioningProfile@1
+        inputs:
+          provisioningProfileLocation: 'secureFiles'
+          provProfileSecureFile: DistributorOnline.mobileprovision
+          removeProfile: true
+
+      - bash: |
+          echo -e "\n[+] Install iOS example dependencies\n"
+          cd example
+
+          echo -e "\n[+] Build ios app\n"
+          npx ionic cap sync ios
+
+          cd ios/App
+
+          echo -e "\n[+] Pod install with --repo-update option \n"
+          pod install --repo-update
+
+        displayName: Install iOS example dependencies
+
+      - bash: |
+
+          echo -e "\n[+] Selecting the Xcode version $(xcodeVersion)\n"
+
+          sudo xcode-select -switch $(xcodeVersion)
+
+          cd example/ios/App/fastlane
+
+          echo -e "\n[+] Setting Apple connection options"
+
+          cp $(api_key.secureFilePath) .
+
+          echo -e "\n[+] Install fastlane \n"
+          gem install fastlane
+
+          export WHATS_NEW="Plugin version: $(pluginVersion), USER: ${{parameters.XAPIEMAIL}}, BUILDINGID: ${{ parameters.BUILDINGID }}, DASHBOARD_URL: https://dashboard.situm.com MAPVIEWER_DOMAIN: ${{ parameters.MAPVIEWERDOMAIN}}"
+          export BUILDNUMBER=${{ parameters.BUILDNUMBER }}
+          export appleDevTeamId=$(appleDevTeamId)
+          echo -e "\n[+] Setting variables to deploy:"
+          echo -e "\t[+] Set WHATS_NEW: $WHATS_NEW"
+
+
+          echo -e "\n[+] Build example with fastlane \n"
+          fastlane ios beta
+
+        displayName: Build iOS example
+
+    - ${{ if contains(parameters.system, 'ubuntu') }}:
+      - task: JavaToolInstaller@0
+        inputs:
+          versionSpec: '17'
```

Comment:
este tipo de cousas se as podes meter en varibles despois é máis doado facer o cambio que aplique a todo